packages/ui/src/components/va-input/VaInput.vue

```diff
@@ -158,6 +158,14 @@ export default defineComponent({
       emit('cleared')
     }
 
+    const focus = () => {
```

Comment:
Minor. This should be on top of useValidation, because it used as callbacks here.

Usually, it's better to define function on top of its usage.

Reply:
Done